### ISSUE TITLE
Add Ljubljana high-res orthophotos

### ISSUE THREAD

**Issue by matheusgomesms** [osmlab/editor-layer-index#2220]
Per discussion: https://community.openstreetmap.org/t/who-can-make-the-ljubljana-orthophoto-available-in-id/108171/12

Still figuring it out the license, but opening issue here for following PR.

URL: [https://gis06.ljubljana.si/arcgis/services/Podloge/DOF1_2023_D96/MapServer/WmsServer?FORMAT=image/png&TRANSPARENT=FALSE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}](https://gis06.ljubljana.si/arcgis/services/Podloge/DOF1_2023_D96/MapServer/WmsServer?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&CRS=%7Bproj%7D&WIDTH=%7Bwidth%7D&HEIGHT=%7Bheight%7D&BBOX=%7Bbbox%7D)

**Comment by cicku** [comment]
Cannot just submit a link to the layer without mentioning the copyright. There are many stuff that you can find on the Internet, and just because you can freely access it does not mean you can use it for OSM.

This also applies to JOSM, which is the main difference between them and us. ELI will check the copyright status very carefully.

glavna.pisarna@ljubljana.si appears to be the contact, since I do not know anything about their language, if you are from the region please reach out and clarify on the copyright first.

**Comment by matheusgomesms** [comment]
Yes, I know. I opened the issue for future reference (and to help me in the future with the URL); the folks from Ljubljana are looking for permission. When (and if) they have permission, I'll create a PR and close this issue.

Thanks for the e-mail, I'll pass to them!
